internal/config/config.go

```diff
@@ -634,3 +636,24 @@ func ValidateArtifacts(artifacts Artifacts) error {
 	}
 	return nil
 }
+
+func ValidatePackageLock() error {
+	_, err := os.Stat("package-lock.json")
+	if err != nil {
+		return fmt.Errorf("missing package-lock.json")
+	}
+	return nil
+}
+
+func ValidatePackage(p node.Package, frameworkName string, expectedVersion string) error {
+	var ver string
+	var ok bool
+	ver, ok = p.Dependencies[frameworkName]
+	if !ok {
+		ver, ok = p.DevDependencies[frameworkName]
+	}
+	if !ok || (expectedVersion != "package.json" && expectedVersion != ver) {
+		return fmt.Errorf("framework version mismatch. The framework version in your config file (%s) must exactly match the framework version in your package.json (%s)", expectedVersion, ver)
+	}
+	return nil
+}
```

Comment:
Would these be better at home in the `node` package? 🤔

Reply:
Could do. The config package has other validation functions. I'm fine with either so if you prefer it in `node`, I can move it.